tools/run_tests/run_tests.py

```diff
@@ -379,14 +379,18 @@ def test_specs(self):
                             if not test: continue
                             cmdline = [binary, '--benchmark_filter=%s$' % test
                                       ] + target['args']
+                            timeout_seconds = _DEFAULT_TIMEOUT_SECONDS * timeout_scaling
```

Comment:
I don't think this is the right place to increase the timeout, run_tests.py is a test runner, it shouldn't double as a place for configuring the test. Isn't there a better place in build.yaml or the scripts that generate tests.json?

Reply:
Interestingly, 
`bm_fullstack_trickle`, seems to be set to `"timeout_seconds": 1200` in 
https://raw.githubusercontent.com/grpc/grpc/master/tools/run_tests/generated/tests.json

Reply:
as a side note: The way the timeouts are set for C/C++ tests is not transparent at all and there is no easy way to see what a timeout for a given test (and configuration) would be - 
I updated timeouts for several tests in the past and it was always a big pain (and it was difficult to check if it was done correctly). I think we will continue having problems with setting test timeout until we do some cleanup a figure out a better approach for this.

Reply:
The source for the timeout seems to be here: 
https://github.com/grpc/grpc/blob/1c4d410c109157bfe924144bc777e57169d5c1e3/build.yaml#L3689

Reply:
Fixed